src/precomputed-requestor.ts

```diff
@@ -25,7 +25,7 @@ export default class PrecomputedFlagRequestor {
 
     await hydrateConfigurationStore(this.precomputedFlagStore, {
       entries: precomputedResponse.flags,
-      environment: precomputedResponse.environment,
+      environment: precomputedResponse.environment ?? { name: '' }, // NOTE: not sure wha the right default to have is here...
```

Comment:
Per Oleksii's notes, we may have an undefined `Environment` in the response. What should the default be?

Reply:
Any reasonable non-empty string sounds good.

Reply:
Edge function is returning `null` environment in this case

Reply:
I put 'UNKNOWN' as a sentinel